packages/material-ui/src/Popper/Popper.test.js

```diff
@@ -110,10 +109,15 @@ describe('<Popper />', () => {
         </ThemeProvider>,
       );
       expect(renderSpy.args).to.deep.equal([['bottom'], ['bottom']]);
-      popperRef.current.options.onUpdate({
-        placement: 'top',
-      });
-      expect(renderSpy.args).to.deep.equal([['bottom'], ['bottom'], ['top'], ['top']]);
+      await popperRef.current.setOptions({ placement: 'top' });
+      expect(renderSpy.args).to.deep.equal([
```

Comment:
This doesn't really test anything. I'll investigate if we can write a better test.